TAF/testScenarios/integrationTest/UC_north_south_messaging/GET-Negative.robot

```diff
@@ -0,0 +1,60 @@
+*** Settings ***
+Resource        TAF/testCaseModules/keywords/common/commonKeywords.robot
+Resource        TAF/testCaseModules/keywords/device-sdk/deviceServiceAPI.robot
+Suite Setup     Setup Suite
+Suite Teardown  Run Teardown Keywords
+Force Tags      Skipped
+
+*** Variables ***
+${SUITE}          North-South Messaging GET Negative Testcases
+${LOG_FILE_PATH}  ${WORK_DIR}/TAF/testArtifacts/logs/north-south-messaging-get-negative.log
+
+*** Test Cases ***
+ErrNSMessagingGET001 - Query Commands with non-existent device name
+    Given Subscribe MQTT Topic 'edgex/commandquery/response'
+    When Get Command With non-existent device From External MQTT Broker
+    Then Should Return Error Code "1"
+    And RequestID Should Be Correct
+
+ErrNSMessagingGET002 - Get specified device non-existent read command
+    Given Create Device For device-virtual
+    And Subscribe MQTT Topic 'edgex/commandquery/response'
+    When Get Command With non-existent command From External MQTT Broker
+    Then Should Return Error Code "1"
+    And RequestID Should Be Correct
+    [Teardown]  Delete device by name
+
```

Comment:
Add a scenario of querying all device commands with invalid offset or limit parameter

Reply:
Updated.